### PR TITLE
Added cheatheet for the idris emacs-mode.

### PR DESCRIPTION
**Pull request by jfdm**
# Cheat sheets for Idris REPL and Emacs modes.

**What does your Instant Answer do?**

A cheatsheet for the Emacs mode for the Idris programming language.

**What is the data source for your Instant Answer? (Provide a link if possible)**

The data source is the source for the emacs mode

https://github.com/idris-hackers/idris-mode

**Why did you choose this data source?**

These are the official sources of information.

**Are there any other alternative (better) data sources?**

No.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

For developers, researchers, and enthusiasts of the dependently typed language Idris.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

Not that I am aware of.

**Are you having any problems? Do you need our help with anything?**

No.

**Where did you hear about DuckDuckHack? (For first time contributors)**

From colleagues, you social media links, and the web.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
Various Screen shots showing collapsed and full views of the cheatsheets.
screen shot 2015-08-01 at 16 40 50
screen shot 2015-08-01 at 17 19 58

-----
IA Page: https://duck.co/ia/view/idris_emacs_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@jfdm thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: share/goodie/cheat_sheets/idris-emacs.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        "Loading Keys",
+        "Documentation",
+        "Editing",
+        "General Keys",
+        "Manipulate Active Terms",
+        "iPKG Keys"
+    ],
+    "sections": {
+        "Loading Keys": [
+            { "key" : "C-c C-l", "val" : "Load File"},
```
Comment (moollaza) [inline share/goodie/cheat_sheets/idris-emacs.json]: Let's wrap each of the keypresses in `[]` so they're visually separated:

`{ "key" : "C-c C-l", "val" : "Load File"},` => `{ "key" : "[C-c] [C-l]", "val" : "Load File"},`

Also, if you wouldn't mind, I'd prefer if this file was formatted like the others, where the keys and values are on separate lines:

```json
{
    "key" : "[C-c] [C-l]",
    "val" : "Load File"
},
{
    "key" : "[C-c] [C-n]",
    "val" : "Load Line Forward"
}, 
```

This is how we've handled things in the Emacs Cheat Sheet as well: https://duckduckgo.com/?q=emacs+cheat+sheet&ia=cheatsheet

**Comment by jfdm** [comment]
Cheers for the review, will update ASAP.

**Comment by moollaza** [comment]
@jfdm congrats! It's live: https://duckduckgo.com/?q=Idris%20emacs%20cheat%20sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**